### DISCUSSION TITLE
List of Creative Agencies / Freelancers who specialize in design using Tailwind?

### DISCUSSION THREAD

**Discussion by aoandrew** [tailwindlabs/tailwindcss#9330] [Help]
Hello @adamwathan and Tailwind community - wondering if there is a curated list of trusted list of agencies/and or freelancers that do UI/UX work using Tailwind?

Thanks!  

I'm most interested in the creative side (as opposed to the technology implementation side.)

**Comment by babakfp** [comment]
Hi

For UI/UX join a few (only design) communities and you will see people sharing their portfolios all over the place. You can also find people in places like dribble or etc. You need to make sure that they design real projects, not fake/imaginary ones.

For Tailwindcss you can join its community and you can also join some front-end communities. If you are looking for a React developer, join a React community and etc. Look for people that know Tailwind very well and used it before. Also, look for people that care about design (not people that only know the technical stuff). You can ask people for portfolios...
